pantab/tests/test_reader.py

```diff
@@ -15,6 +16,19 @@ def test_read_doesnt_modify_existing_file(df, tmp_hyper):
     assert last_modified == tmp_hyper.stat().st_mtime
 
 
+def test_reports_unsupported_type():
+    """
+    Test that we report an error if we encounter an unsupported column type.
+    Previously, we did not do so but instead assumed that all unsupported columns
+    would be string columns. This led to very fascinating failures.
+    """
+    db_path = pathlib.Path(__file__).parent / "geography.hyper"
```

Comment:
I think you mean to commit this file right? Can you rebase on top of or merge master and add this to the data folder in the tests directory?

Reply:
Right...
The `.gitignore` entry for all Hyper files caught me here :/